code/modules/plumbing/plumbers/plumbing/input_and_output.dm

```diff
@@ -0,0 +1,33 @@
+/obj/machinery/plumbing/pipeinput
+	name = "pipe input"
```

Comment:
You call these things input pipes everywhere else. Would be more consistent if you also did it here.